spec/integration/fixtures/overwrite/array_in_hash_pattern_transpiled.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+using RubyNext;
```

Comment:
We can't store a once transpiled file as a fixture, because the transpilation result is likely to change in the future.

Alternatively, we might create a transpiled file first (without `--overwrite` option) and compare it with overwritten one.